README.md

```diff
@@ -1,5 +1,9 @@
-statsd_exporter
-=============
+# statsd exporter [![Build Status](https://travis-ci.org/prometheus/statsd_exporter.svg)][travis]
+
+[![CircleCI](https://circleci.com/gh/prometheus/statsd_exporter/tree/master.svg?style=shield)][circleci]
+[![Image Size & Layers](https://imagelayers.io/badge/prom/statsd-exporter:master.svg)][imagelayers]
+[![Docker Stars](https://img.shields.io/docker/stars/prom/statsd-exporter.svg)][hub]
+[![Docker Pulls](https://img.shields.io/docker/pulls/prom/statsd-exporter.svg)][hub]
```

Comment:
The shields discussion is [still going](https://github.com/prometheus/prometheus/pull/1598), but I think we should replace the `imagelayer` shield by a Quay link and remove the `Docker Stars`.

```
[![Docker Repository on Quay](https://quay.io/repository/prometheus/statsd-exporter/status)][quay]

[quay]: https://quay.io/repository/prometheus/statsd-exporter
```